Makefile

```diff
@@ -277,7 +277,7 @@ endif
 # Userguide HTML docs are copied to dist/userguide
 userguide: $(USERGUIDE_TARGET_DIR) $(USERGUIDE_ZIP_FILE)
 $(USERGUIDE_TARGET_DIR): $(GIT_UG_REPO_PATH) $(GIT_SAMPLE_DATA_REPO_PATH) | $(DIST_DIR)
-	ln -s $(WORKING_DIR)/$(GIT_SAMPLE_DATA_REPO_PATH) $(WORKING_DIR)/$(GIT_UG_REPO_PATH)
+	-ln -s $(WORKING_DIR)/$(GIT_SAMPLE_DATA_REPO_PATH) $(WORKING_DIR)/$(GIT_UG_REPO_PATH)
```

Comment:
I believe it indicates that the exit status of that command can be ignored by `make`. Normally if a command exited with an error code, that would break the execution of the rest of the `make` target. But in my case I somehow had a pre-existing softlink which caused this to raise an error when it tried making a new one. So I thought it okay to tell `make` to ignore an error from `ln`.

Reply:
Ah okay thanks for explaining!